### PR TITLE
Update AbstractRanges

### PR DESCRIPTION
**Pull request by teoli2003**
#### Summary
This is part of the DOM project ( #9740 )

It updates `AbstractRange` to the latest standard.

Note this is an abstract class, so more information will be in the descendant of these classes.

### CODE REVIEW

File: files/en-us/web/api/abstractrange/index.md [hunk 1]
```diff
@@ -2,52 +2,45 @@
...
-As an abstract interface, you will not directly instantiate an object of type `AbstractRange`. Instead, you will use the {{domxref("Range")}} or {{domxref("StaticRange")}} interfaces. To understand the difference between those two interfaces, and how to choose which is appropriate for your needs.
+> **Note:** As an abstract interface, you will not directly instantiate an object of type `AbstractRange`. Instead, you will use the {{domxref("Range")}} or {{domxref("StaticRange")}} interfaces. To understand the difference between those two interfaces, and how to choose which is appropriate for your needs.
 
 {{InheritanceDiagram}}
 
 ## Properties
 
 - {{domxref("AbstractRange.collapsed", "collapsed")}} {{ReadOnlyInline}}
-  - : A Boolean value which is `true` if the range is **collapsed**. A collapsed range is one whose start position and end position are the same, resulting in a zero-character-long range.
+  - : A Boolean value which is `true` if the range is _collapsed_. A collapsed range is a rangte whose start position and end position are the same, resulting in a zero-character-long range.
```
Comment (wbamberg) [inline files/en-us/web/api/abstractrange/index.md:20]:
```suggestion
  - : A Boolean value which is `true` if the range is _collapsed_. A collapsed range is a range whose start position and end position are the same, resulting in a zero-character-long range.
```

File: files/en-us/web/api/abstractrange/index.md [hunk 2]
```diff
@@ -2,52 +2,45 @@
...
+  - : An integer value indicating the offset, in characters, from the beginning of the node's contents to the beginning of the range represented by the range object. This value must be less than the length of the `endContainer` node.
 - {{domxref("AbstractRange.startContainer", "startContainer")}} {{ReadOnlyInline}}
-  - : The DOM {{domxref("Node")}} in which the beginning of the range, as specified by the `startOffset` property, is located.
+  - : The DOM {{domxref("Node")}} in which the beginning of the range, as specified by the `startOffset` property, is located.
 - {{domxref("AbstractRange.startOffset", "startOffset")}} {{ReadOnlyInline}}
-  - : An integer value indicating the offset, in characters, from the beginning of the node's contents to the last character of the contents referred to  by the range object. This value must be less than the length of the node indicated in `startContainer`.
+  - : An integer value indicating the offset, in characters, from the beginning of the node's contents to the last character of the contents referred to  by the range object. This value must be less than the length of the node indicated in `startContainer`.
```
Comment (wbamberg) [inline files/en-us/web/api/abstractrange/index.md:28]:
```suggestion
  - : An integer value indicating the offset, in characters, from the beginning of the node's contents to the last character of the contents referred to by the range object. This value must be less than the length of the node indicated in `startContainer`.
```

File: files/en-us/web/api/abstractrange/collapsed/index.md [hunk 3]
```diff
@@ -2,30 +2,24 @@
...
-var isCollapsed = range.collapsed
-```
+A boolean value which is `true` if the range is _collapsed_. A collapsed range is one in which the start and end positions are the same, resulting in a zero-character-long range.
 
-### Value
+## Example
 
-A boolean value which is `true` if the range is **collapsed**. A collapsed range is one in which the start and end positions are the same, resulting in a zero-character-long range.
+```js
+let isCollapsed = range.collapsed
```
Comment (wbamberg) [inline files/en-us/web/api/abstractrange/collapsed/index.md:21]: Some of these have  semicolons at the end and some don't I think if these are in "Examples" they should probably all have semicolons.

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks @teoli2003 ! Just a few copy-edit type things.

File: files/en-us/web/api/abstractrange/collapsed/index.md (hunk 3 above)
Comment (teoli2003) [reply files/en-us/web/api/abstractrange/collapsed/index.md]: Added.

**Comment by teoli2003** [comment]
Good catch for the semi-colons in one-line example!

File: files/en-us/web/api/abstractrange/index.md [hunk 4]
```diff
@@ -124,7 +124,7 @@ Here an interesting problem arises—we are capturing content from multiple node
 
 As it turns out, the DOM specification fortunately addresses this exact issue. For example, in this case, we're calling {{domxref("Range.cloneContents", "cloneContents()")}} on the range to create a new {{domxref("DocumentFragment")}} object providing a DOM subtree which replicates the contents of the specified range. To do this, `cloneContents()` constructs all the nodes needed to preserve the structure of the indicated range, but no more than necessary.
 
-In this example, the start of the specified range is found within the text node below the section's heading, which means that the new `DocumentFragment` will need to contain an {{HTMLElement"Heading_Elements", ("h2")}} and, below it, a text node.
+In this example, the start of the specified range is found within the text node below the section's heading, which means that the new `DocumentFragment` will need to contain an {{HTMLElement("Heading_Elements", ("h2")}} and, below it, a text node.
```
Comment (wbamberg) [inline files/en-us/web/api/abstractrange/index.md:127]:
```suggestion
In this example, the start of the specified range is found within the text node below the section's heading, which means that the new `DocumentFragment` will need to contain an {{HTMLElement("Heading_Elements", "h2")}} and, below it, a text node.
```

**Review by wbamberg — APPROVED** [review]
👍 thanks !

**Pull request merged**